src/software/ai/hl/stp/tactic/move/move_fsm.h

```diff
@@ -79,6 +79,7 @@ struct MoveFSM
             // src_state + event [guard] / action = dest_state
             *move_s + update_e[!move_done] / update_move = move_s,
             move_s + update_e[move_done] / update_move   = X,
-            X + update_e[!move_done] / update_move       = move_s);
+            X + update_e[!move_done] / update_move       = move_s,
+            X + update_e[move_done] / update_move        = X);
```

Comment:
A cool benefit of the FSMs. One of the theta start tests was failing because the robot was reaching it's threshold for "arrived at destination" and then coasting _juuuust_ outside the bounding box the test was checking. Rather than changing the bounding box, I decided to have the MoveFSM continually move to it's destination, but it will still be "done" once `move_done` returns false.